src/proto/flwr/proto/appio.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
```

Comment:
This file should be named `clientappio.proto` (the might be a `serverappio.proto` in the future)